src/pymmcore_widgets/hcs/_fov_widget/_fov_sub_widgets.py

```diff
@@ -0,0 +1,768 @@
+    QGraphicsScene,
+    QGroupBox,
+    QLabel,
+    QPushButton,
+    QSizePolicy,
+    QSpacerItem,
+    QSpinBox,
+    QVBoxLayout,
+    QWidget,
+)
+from useq import GridRowsColumns, Position, RandomPoints, RelativePosition
+from useq._grid import OrderMode, Shape
+
+from pymmcore_widgets.hcs._graphics_items import (
+    FOV,
+    GREEN,
+    _FOVGraphicsItem,
+    _WellAreaGraphicsItem,
+)
+from pymmcore_widgets.hcs._util import _ResizingGraphicsView, nearest_neighbor
+
+from ._util import create_label, make_wdg_with_label
+
+AlignCenter = Qt.AlignmentFlag.AlignCenter
+FIXED_POLICY = (QSizePolicy.Policy.Fixed, QSizePolicy.Policy.Fixed)
+DEFAULT_VIEW_SIZE = (300, 300)  # px
+DEFAULT_WELL_SIZE = (10, 10)  # mm
+DEFAULT_FOV_SIZE = (DEFAULT_WELL_SIZE[0] / 10, DEFAULT_WELL_SIZE[1] / 10)  # mm
+PEN_WIDTH = 4
+RECT = Shape.RECTANGLE
+ELLIPSE = Shape.ELLIPSE
+PEN_AREA = QPen(QColor(GREEN))
+PEN_AREA.setWidth(PEN_WIDTH)
+
+
+class Center(Position):
+    """A subclass of GridRowsColumns to store the center coordinates and FOV size.
+
+    Attributes
+    ----------
+    fov_width : float | None
+        The width of the FOV in µm.
+    fov_height : float | None
+        The height of the FOV in µm.
+    """
+
+    fov_width: Optional[float] = None  # noqa: UP007
+    fov_height: Optional[float] = None  # noqa: UP007
+
+
+class _CenterFOVWidget(QWidget):
+    """Widget to select the center of a specifiied area."""
+
+    def __init__(self, parent: QWidget | None = None) -> None:
+        super().__init__(parent)
+
+        self._x: float = 0.0
+        self._y: float = 0.0
+
+        self._fov_size: tuple[float | None, float | None] = (None, None)
+
+        lbl = QLabel(text="Center of the Well.")
+        lbl.setStyleSheet("font-weight: bold;")
+        lbl.setAlignment(AlignCenter)
+
+        # add widgets layout
+        self.wdg = QGroupBox()
+        layout = QVBoxLayout(self.wdg)
+        layout.setSpacing(0)
+        layout.setContentsMargins(10, 10, 10, 10)
+        layout.addWidget(lbl)
+
+        # main
+        main_layout = QVBoxLayout(self)
+        main_layout.setSpacing(0)
+        main_layout.setContentsMargins(0, 0, 0, 0)
+        main_layout.addWidget(self.wdg)
+
+    @property
+    def fov_size(self) -> tuple[float | None, float | None]:
+        """Return the FOV size."""
+        return self._fov_size
+
+    @fov_size.setter
+    def fov_size(self, size: tuple[float | None, float | None]) -> None:
+        """Set the FOV size."""
+        self._fov_size = size
+
+    def value(self) -> Center:
+        """Return the values of the widgets."""
+        fov_x, fov_y = self._fov_size
+        return Center(x=self._x, y=self._y, fov_width=fov_x, fov_height=fov_y)
+
+    def setValue(self, value: Center) -> None:
+        """Set the values of the widgets."""
+        self._x = value.x or 0.0
+        self._y = value.y or 0.0
+        self.fov_size = (value.fov_width, value.fov_height)
+
+
+class _RandomFOVWidget(QWidget):
```

Comment:
might make more sense in the `useq_widgets` module.  since it deals with creation of a pure useq object